SpriterDotNet/Parser/XmlSpriterParser.cs

```diff
@@ -24,7 +26,14 @@ public Spriter Parse(string data)
 
         public bool CanParse(string data)
         {
-            return data.StartsWith(XmlStart);
+            return data != null && data.StartsWith(XmlStart);
```

Comment:
We're already checking the data for null in the SpriterParser before iterating through the parsers so we can assume the data is not null and trimmed at this point.